src/scout_apm/bottle/__init__.py

```diff
@@ -1,26 +1,25 @@
-from __future__ import absolute_import
+from __future__ import absolute_import, division, print_function, unicode_literals
 
 from bottle import request
 
 import scout_apm.core
-from scout_apm.core.context import AgentContext
+from scout_apm.api.context import Context
```

Comment:
These are not the same. Unfortunately we use the word `Context` in both modules.

`AgentContext` is for `scout_apm` internal/core use only and provides a `state of the agent`.

`Context` provides a way for our instrumentation and end-users to attach bits of information to a `TrackedRequest`

Reply:
Oops, I see this was just an ordering change. It was confusing to see in the GitHub diff. Ignore.